get-started/index.md

```diff
@@ -652,15 +652,15 @@ spec:
 ```
 
 ## [Next Steps](#next-steps)
-Now that you have installed and deployed Cryostat and know how to access its
-web client, continue on to [Guides]({% link guides/index.md %}) for
+Now that you have installed and deployed **Cryostat** and know how to access its
+**web client**, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
 
 ## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
-Reference [Operator Lifecycle Manager's](https://olm.operatorframework.io/docs/tasks/uninstall-operator/#combine-steps-2-and-3)
-guide on uninstalling Operators. Please be sure to delete all `Cryostat` and `ClusterCryostat` Custom Resources before
-uninstalling the Cryostat Operator.
-- If your Cryostat Operator was installed in **All Namespaces** mode, then its `ClusterServiceVersion` and
-`Subscription` can be found in the `Namespace` `openshift-operators`.
-- If your Cryostat Operator was installed in **A specific Namespace**, then the `ClusterServiceVersion` and
-`Subscription` will be in that same `Namespace.`
+Reference [**Operator** Lifecycle Manager's](https://olm.operatorframework.io/docs/tasks/uninstall-operator/#combine-steps-2-and-3)
```

Comment:
**OLM** rather than "**Operator** Lifecycle Manger" - again, we have been using "**Operator**" as shorthand for "**Cryostat Operator**". Operator Lifecycle Manager is its own separate thing.